source/h2_stream.c

```diff
@@ -58,6 +59,123 @@ static struct aws_h2_connection *s_get_h2_connection(const struct aws_h2_stream
 #define AWS_PRECONDITION_ON_CHANNEL_THREAD(STREAM)                                                                     \
     AWS_PRECONDITION(aws_channel_thread_is_callers_thread(s_get_h2_connection(STREAM)->base.channel_slot->channel))
 
+static bool s_client_state_allows_frame_type[AWS_H2_STREAM_STATE_COUNT][AWS_H2_FRAME_T_UNKNOWN] = {
```

Comment:
Nitpick: Could we add an ```AWS_H2_FRAME_T_COUNT = AWS_H2_FRAME_T_UNKNOWN``` to that enum to make usages like this less unintuitive?

Reply:
Yeah, I'll add that